client/src/app/containers/Suggestions/SuggestionsTable.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+
+import Grid from '@material-ui/core/Grid';
+import GridList from '@material-ui/core/GridList';
+import Paper from '@material-ui/core/Paper';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import ChoosingCard from 'app/containers/Suggestions/ChoosingCard';
+
+import { selectSuggestions } from 'app/containers/Suggestions/selectors';
+import { sliceKey, reducer } from 'app/containers/Suggestions/slice';
+import { useSelector } from 'react-redux';
+import { useInjectReducer, useInjectSaga } from 'redux-injectors';
+
+const SuggestionsTable = () => {
+  const sugs = useSelector(selectSuggestions);
+
+  if (!Array.isArray(sugs)) {
+    return (`suggestions are: ${sugs}`);
+  }
+
+  return (
+    <TableContainer component={Paper}>
+      <Table stickyHeader size='small' aria-label='Hand Suggestions table'>
+        <TableHead>
+          <TableRow>
+            <TableCell>Hand Points (avg)</TableCell>
+            <TableCell>Hand Points (median)</TableCell>
+            <TableCell>Crib Points (avg)</TableCell>
+            <TableCell>Crib Points (median)</TableCell>
+            <TableCell>Throw</TableCell>
+          </TableRow>
```

Comment:
It would be awesome if we could get a table that has "merged cells" for the first header row, then has sub columns in the second header row so that we put `| hand points | crib points | throw |` in the first row and have `avg | median | max` as "sub columns" under each hand and crib. I haven't been able to figure that out yet, but it seems like [this page](https://material-ui.com/components/tables/) should be able to help?